oops.min.js

```diff
@@ -0,0 +1,115 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-var */
+/* eslint-disable max-len */
+
+// eslint-disable-next-line require-jsdoc
+function Oops(a) {
```

Comment:
- [ ] The form data for all inputs should be stored in a single object, and saved to a single local storage registry, please implement this feature so your code meets the requirements